pubsublite/driver/driver.go

```diff
@@ -0,0 +1,9 @@
+package driver
+
+import "context"
+
+type Pubsubber interface {
+	Publish(ctx context.Context, topic string, msg []byte) error
+	Subscribe(ctx context.Context, callback func(msg []byte)) error
```

Comment:
For NATS we like callbacks to have Subject, Reply and Data, which we wrap up in a nats.Msg.

Since we can subscribe to wildcards it may be important to evaluate the literal subject it was published on. Reply is for us to reply by Publishing the response.

Reply:
I'm not sure such things should be exposed at this level. I would think there could be an RPC layer on top of raw pubsublite, and the RPC layer would know about Subject, Reply and Data and take care of encoding and decoding.

Reply:
Possibly, although seems a better fit here IMO. As for subject, at least for NATS that one can be important since you can subscribe to `foo.bar.*` but the messages will be delivered on literals, like `foo.bar.baz` which the user may need to know to process the message.

Reply:
Okay, I think we can handle this for now by just making these take an Msg struct with a Data field. Then we can add the other fields as needed.